libs/ett/core/shell/src/lib/ett-core-shell.module.ts

```diff
@@ -20,18 +20,21 @@ import {
   EttAuthenticationGuard,
   ettAuthRoutePath,
 } from '@energinet-datahub/ett/auth/routing-security';
-import { ettDashboardRoutePath } from '@energinet-datahub/ett/dashboard/routing';
 import { GfBrowserConfigurationModule } from '@energinet-datahub/gf/util-browser';
 
 import { EttHttpModule } from './ett-http.module';
 import { EttMaterialModule } from './ett-material.module';
 import { EttShellComponent, EttShellScam } from './ett-shell.component';
+import { ettDashboardRoutePath } from '@energinet-datahub/ett/dashboard/routing';
```

Comment:
issue: Move this import statement back to where it was to sort imports alphabetically and by type.